.eslintrc.js

```diff
@@ -47,5 +48,29 @@ module.exports = {
     'no-var': 'error',
     'prefer-rest-params': 'error',
     'prefer-spread': 'error',
+    "jest/expect-expect": [
+      "error",
+      {
+        // we set this to `expect*` so that any function whose name starts with expect will be counter
+        // as an assertion function, allowing us to use functions to DRY up test suites.
+        "assertFunctionNames": ["expect*"],
+      }
+    ],
+    // we have a number of things disabled w/ `xdescribe`, `xit`, etc
+    "jest/no-test-prefixes": ["off"],
```

Comment:
👍 cool that makes sense to me, I started to convert them over but backed out because I wasn't sure if `xit` was sort of part of an informal style guide, I also find `.skip` to be more obvious and also just feels more 'jest-ey' if you will

Reply:
made the change here: https://github.com/ionic-team/stencil/pull/3447/commits/315bc31b293a4560886fce6f153ba547b042a5cf